utils/utils.jsx

```diff
@@ -1073,6 +1073,16 @@ export function displayUsernameForUser(user) {
     return '';
 }
 
+/**
+ * Sort users according to display name, respecting the TeammateNameDisplay configuration setting
+ */
+export function sortUsersByDisplayName(a, b) {
+    const aName = displayUsernameForUser(a);
+    const bName = displayUsernameForUser(b);
+
+    return aName.localeCompare(bName);
+}
```

Comment:
Perhaps accept two user objects here, and do all the sorting, including the status weighting, so that the components can just call this function?